NightscoutUploadKit/NightscoutUploader.swift

```diff
@@ -169,7 +169,7 @@ public class NightscoutUploader: NSObject {
                 "timestamp": pumpDateStr,
                 "bolusiob": status.iob,
             ],
-            "reservoir": status.reservoirRemainingUnits,
+            "reservoir": Double(round(status.reservoirRemainingUnits*1000)/1000,
```

Comment:
Well, it manifests like this: 0.07500000000000001 instead of 0.075.  Not a serious issue in terms of accuracy.